v-next/hardhat/src/internal/builtin-plugins/network-manager/edr/utils/console-logger.ts

```diff
@@ -0,0 +1,267 @@
+import util from "node:util";
+
+import { assertHardhatInvariant } from "@ignored/hardhat-vnext-errors";
+import {
+  bytesToHexString as bufferToHex,
+  bytesToHexString,
+} from "@ignored/hardhat-vnext-utils/bytes";
+import { hexStringToBigInt } from "@ignored/hardhat-vnext-utils/hex";
+
+import {
+  AddressTy,
+  BoolTy,
+  Bytes10Ty,
+  Bytes11Ty,
+  Bytes12Ty,
+  Bytes13Ty,
+  Bytes14Ty,
+  Bytes15Ty,
+  Bytes16Ty,
+  Bytes17Ty,
+  Bytes18Ty,
+  Bytes19Ty,
+  Bytes1Ty,
+  Bytes20Ty,
+  Bytes21Ty,
+  Bytes22Ty,
+  Bytes23Ty,
+  Bytes24Ty,
+  Bytes25Ty,
+  Bytes26Ty,
+  Bytes27Ty,
+  Bytes28Ty,
+  Bytes29Ty,
+  Bytes2Ty,
+  Bytes30Ty,
+  Bytes31Ty,
+  Bytes32Ty,
+  Bytes3Ty,
+  Bytes4Ty,
+  Bytes5Ty,
+  Bytes6Ty,
+  Bytes7Ty,
+  Bytes8Ty,
+  Bytes9Ty,
+  BytesTy,
+  Int256Ty,
+  StringTy,
+  Uint256Ty,
+  CONSOLE_LOG_SIGNATURES,
+} from "./console-log-signatures.js";
+
+/**
+ * TODO: This is pulled from v-next utils. I don't understand why it returns a `number | bigint`
+ */
+function bytesToBigInt(bytes: Uint8Array): bigint {
```

Comment:
This can be replaced by the version in hh-utils once the latest update is merged with the fixed return type.